.github/workflows/test-python.yml

```diff
@@ -65,22 +69,22 @@ jobs:
           net start MongoDB
       - name: Install libbson
         run: |
-          LIBBSON_INSTALL_DIR=$(pwd)/libbson ./build-libbson.sh
+          ./build-libbson.sh
       - name: Install Python dependencies
         run: |
           python -m pip install -U pip
       - name: Install pymongoarrow
         run: |
           # Install the library with no deps
-          LIBBSON_INSTALL_DIR=$(pwd)/libbson python -m pip install -vv -e "."
+          python -m pip install -vv -e "."
       - name: Ensure imports with no test deps
         run: |
           python -c "from pymongoarrow.monkey import patch_all; patch_all()"
       - name: Run tests
         run: |
           # Install the test deps
-          LIBBSON_INSTALL_DIR=$(pwd)/libbson python -m pip install -vv -e ".[test]"
-          PYTHONWARNINGS=error LD_LIBRARY_PATH=$(pwd)/libbson/lib python -m unittest discover test -v
+          python -m pip install -vv -e ".[test]"
```

Comment:
The `-vv` here and above causes thousands of lines of pip output which makes it difficult to look at the test runs. Can we remove it completely (or use `-v` if that lowers the verbosity)? Or maybe we can use `-v` for the "Install pymongoarrow" part and no `-v` for the "Run tests" part?

Reply:
Updated